appConfig.ts

```diff
@@ -0,0 +1,5 @@
+const Config = {
+    API_URL: "http://localhost:3333/dsayling8/ZoomFoodToo/1.0.0"
```

Comment:
should probably breakup host, port and path to support prod deployments, but keep the path the same

Reply:
Also, I need to remve my username from the URL path